src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
Here's what I've come up with:
```
InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
{

#ifdef DEBUG

    // Optionally install the RandomPolicy.
    // 
    // We do this when stress is enabled and we're not going to enable
    // legacy inline stress (roughly 50% of the time if both inline
    // stress modes are enabled).
    //
    // Thus we have:
    // All legacy: (default), JitStress=2 with all modes enabled
    //  50/50 mix: JitStress != (0,2) with all modes enabled
    // All random: JitStress != 0 with legacy mode not enabled. Random seed
    //   is the jit stress value

    bool useRandomPolicy = compiler->compInlineRandomStress() && !compiler->compInlineStress();

    if (useRandomPolicy)
    {
        unsigned seed = getJitStressLevel();
        assert(seed != 0);
        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
    }

#endif // DEBUG
```

That way I can get my "all random w/varying seed" and we mix in some random behavior during JitStress with values other than 2.